cni/network/invoker_cns.go

```diff
@@ -120,6 +120,7 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 	}
 
 	addResult := IPAMAddResult{}
+	var isDefaultInterfaceSet bool // make sure isDefaultInterface is set accordingly for defaultCniResult
```

Comment:
I couldn't understand anything from this comment. Please add detailed description how this flag will be set for different scenario

Reply:
updated comment but we can also just default interface value in network.go if there are no secondary CNIResults